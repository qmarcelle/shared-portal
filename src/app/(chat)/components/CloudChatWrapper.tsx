'use client';

// CloudChatWrapper injects the Genesys Cloud Messenger SDK and manages Messenger-specific events.
// All script loading, Messenger events, and errors are logged for traceability and debugging.

import { useEffect } from 'react';
import { useChatStore } from '../stores/chatStore';

export default function CloudChatWrapper() {
  const { userData } = useChatStore();

  useEffect(() => {
    // Build chatSettings from env and userData
    window.chatSettings = {
      bootstrapUrl: process.env.NEXT_PUBLIC_GENESYS_BOOTSTRAP_URL || '',
      widgetUrl: process.env.NEXT_PUBLIC_GENESYS_WIDGET_URL || '',
      clickToChatJs: process.env.NEXT_PUBLIC_GENESYS_CLICK_TO_CHAT_JS || '',
      clickToChatEndpoint: process.env.NEXT_PUBLIC_CLICK_TO_CHAT_ENDPOINT || '',
      chatTokenEndpoint: process.env.NEXT_PUBLIC_CHAT_TOKEN_ENDPOINT || '',
      coBrowseEndpoint: process.env.NEXT_PUBLIC_COBROWSE_LICENSE_ENDPOINT || '',
      opsPhone: process.env.NEXT_PUBLIC_OPS_PHONE || '',
      opsPhoneHours: process.env.NEXT_PUBLIC_OPS_HOURS || '',
      ...userData,
    };
    console.debug('[CloudChatWrapper] chatSettings:', window.chatSettings);
  }, [userData]);

  useEffect(() => {
    // Load Genesys Cloud bootstrap and widget scripts
    const bootstrapUrl = window.chatSettings?.bootstrapUrl || '';
    const widgetUrl = window.chatSettings?.widgetUrl || '';
    if (!bootstrapUrl) return;
    const bootstrapScript = document.createElement('script');
    bootstrapScript.src = bootstrapUrl;
    bootstrapScript.async = true;
    bootstrapScript.onload = () => {
      console.debug('[CloudChatWrapper] Genesys bootstrap loaded');
      if (!widgetUrl) return;
      const widgetScript = document.createElement('script');
      widgetScript.src = widgetUrl;
      widgetScript.async = true;
      widgetScript.onload = () => {
        console.debug('[CloudChatWrapper] Genesys widgets loaded');
      };
      widgetScript.onerror = (e) => {
        console.error('[CloudChatWrapper] Failed to load widgets script', e);
      };
      document.body.appendChild(widgetScript);
    };
    bootstrapScript.onerror = (e) => {
      console.error('[CloudChatWrapper] Failed to load bootstrap script', e);
    };
    document.body.appendChild(bootstrapScript);

    // Provide openGenesysChat logic
    window.openGenesysChat = () => {
      if (window.Genesys) {
        try {
          window.Genesys('command', 'Messenger.open');
        } catch (e) {
          console.error(
            '[CloudChatWrapper] Error opening Genesys Messenger',
            e,
          );
        }
      }
    };

    return () => {
      // Cleanup scripts and chatSettings
      document.body.removeChild(bootstrapScript);
      if (widgetUrl) {
        const widgetScript = document.querySelector(
          'script[src="' + widgetUrl + '"]',
        );
        if (widgetScript) document.body.removeChild(widgetScript);
      }
      if (window.chatSettings) delete window.chatSettings;
      if (window.openGenesysChat) delete window.openGenesysChat;
      console.debug('[CloudChatWrapper] Cleanup complete');
    };
  }, [userData]);

  return null;
}
