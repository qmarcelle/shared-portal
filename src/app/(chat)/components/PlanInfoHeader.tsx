'use client';
import { useChatStore } from '@/app/(chat)/stores/chatStore';
import { Title } from '@/components/foundation/Title';

/**
 * Show current plan when multiple plans exist
 */
export default function PlanInfoHeader() {
  const { formInputs } = useChatStore();
  if (formInputs.length <= 1) return null;
  const plan = formInputs.find((f) => f.id === 'PLAN_ID');

  return (
    <Title
      className="plan-info-header"
      text={`You are chatting about plan ${plan?.value || ''}`}
    />
  );
}
