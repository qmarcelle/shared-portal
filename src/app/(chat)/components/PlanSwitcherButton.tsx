'use client';
import { useChatStore } from '@/app/(chat)/stores/chatStore';
import { Button } from '@/components/foundation/Button';
import { usePlanStore } from '../../../userManagement/stores/planStore';

/**
 * "Switch Plan" inside pre-chat window
 */
export default function PlanSwitcherButton() {
  const { endChat } = useChatStore();
  const { plans, selectedPlanId, openPlanSwitcher } = usePlanStore();

  if (plans.length <= 1) return null;

  const handleSwitch = () => {
    endChat();
    openPlanSwitcher();
  };

  return (
    <Button
      label="Switch Plan"
      callback={handleSwitch}
      className="plan-switch-button"
      type="secondary"
    />
  );
}
