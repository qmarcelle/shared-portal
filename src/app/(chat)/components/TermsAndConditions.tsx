'use client';
// import TCs from '@/app/@chat/config/terms'; // { default: '…', lobA: '…', … }
import { useChatStore } from '@/app/(chat)/stores/chatStore';
import { RichText } from '@/components/foundation/RichText';

/**
 * Render T&C based on LOB
 */
export default function TermsAndConditions() {
  const { userData } = useChatStore();
  // const text = TCs[userData.LOB] || TCs.default;
  return (
    <RichText
      spans={['Terms and Conditions']}
      className="terms-and-conditions"
      type="body-2"
    />
  );
}
