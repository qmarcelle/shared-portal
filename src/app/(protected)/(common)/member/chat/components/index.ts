/**
 * Component Exports
 * Centralizes all component exports following the new consolidated structure
 */

// Main Components
export { ChatTrigger } from './ChatTrigger';
export { ChatWidget } from './ChatWidget';
export { PlanInfoHeader } from './PlanInfoHeader';

// Error Handling
export { ChatErrorBoundary } from './shared/ChatErrorBoundary';

// Integration Components (to be consolidated)
export { GenesysScripts } from './GenesysScripts';
