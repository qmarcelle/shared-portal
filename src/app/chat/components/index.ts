// Core Components
export * from './core/ChatButton';
export * from './core/ChatErrorBoundary';
export * from './core/ChatHeader';
export * from './core/ChatWidget';

// Business Hours Components
export * from './business-hours/BusinessHoursNotification';

// Eligibility Components
export * from './eligibility/EligibilityCheck';

// Plan Switcher Components will be added here when implemented
// export * from './plan-switcher/PlanSwitcher';
