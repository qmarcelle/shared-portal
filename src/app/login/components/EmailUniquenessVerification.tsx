import { Button } from '@/components/foundation/Button';
import { Divider } from '@/components/foundation/Divider';
import { Header } from '@/components/foundation/Header';
import { RichText } from '@/components/foundation/RichText';
import { Spacer } from '@/components/foundation/Spacer';
import { TextBox } from '@/components/foundation/TextBox';
import { TextField } from '@/components/foundation/TextField';
import {
  formatDate,
  isConfirmEmailAddressMatch,
  isValidEmailAddress,
  validateDate,
  validateLength,
} from '@/utils/inputValidator';
import { useState } from 'react';

export const EmailUniqueVerification = () => {
  const [invalidDate, setInvalidDate] = useState<string[]>([]);
  const [emailError, setEmailError] = useState<string[]>([]);
  const [emailConfirmationError, setemailConfirmationError] = useState<
    string[]
  >([]);
  const [formEmailData, setFormEmailData] = useState({
    email: '',
    confirmEmail: '',
  });
  const [date, setDate] = useState('');
  let isBackSpacePressed: boolean = false;

  const handleChange = (val: string) => {
    let formattedDate = val;
    if (!isBackSpacePressed) {
      formattedDate = formatDate(val);
    }
    setDate(formattedDate);
    if (!validateDate(date)) {
      setInvalidDate(['Please follow the MM/DD/YYYY format.']);
    } else {
      setInvalidDate([]);
    }
  };

  const confirmEmailAddress = (value: string) => {
    setFormEmailData({
      ...formEmailData,
      ['confirmEmail']: value,
    });
    const isEmailMatch = isConfirmEmailAddressMatch(value, formEmailData.email);
    if (!isEmailMatch) {
      setemailConfirmationError(['Email must match']);
    } else {
      setemailConfirmationError([]);
    }
  };

  const validateEmailAddress = (value: string) => {
    const isValidEmail = isValidEmailAddress(value);
    const isValidLength = validateLength(value);
    setFormEmailData({
      ...formEmailData,
      ['email']: value,
    });
    if (!isValidEmail && !isValidLength) {
      setEmailError(['Invalid Email Address']);
    } else if (isValidEmail && !isValidLength) {
      setEmailError(['Invalid Email Address']);
    } else if (!isValidEmail && isValidLength) {
      setEmailError(['Invalid Email Address']);
    } else {
      setEmailError([]);
    }
    if (
      value !== formEmailData.confirmEmail &&
      formEmailData.confirmEmail !== ''
    ) {
      setemailConfirmationError(['Email must match']);
    } else {
      setemailConfirmationError([]);
    }
  };
  const keyDownCallBack = (keyCode: string) => {
    isBackSpacePressed = keyCode == 'Backspace';
  };

  return (
    <div id="mainSection">
      <Header className="title-2" text="Choose Your Email Address"></Header>
      <Spacer size={16} />
      <TextBox
        className="body-1"
        text="Your email is invalid, or the original email address you provided is already associated with another account. Please provide your date of birth and new email address to continue."
      />
      <Spacer size={32} />
      <TextField
        className="body-1"
        label="Date of Birth (MM/DD/YYYY)"
        valueCallback={(val) => handleChange(val)}
        onKeydownCallback={(val) => keyDownCallBack(val)}
        value={date}
        errors={invalidDate}
        maxLength={10}
      ></TextField>
      <Spacer size={32} />
      <TextField
        className="body-1"
        label="Email Address"
        valueCallback={(val) => validateEmailAddress(val)}
        type="email"
        errors={emailError}
        value={formEmailData.email}
      ></TextField>
      <Spacer size={32} />
      <TextField
        className="body-1"
        label="Confirm Email Address"
        type="email"
        value={formEmailData.confirmEmail}
        valueCallback={(val) => confirmEmailAddress(val)}
        errors={emailConfirmationError}
      ></TextField>
      <Spacer size={32} />
      <Button label="Next" />
      <Spacer size={32} />
      <Divider />
      <Spacer size={32} />
      <TextBox className="title-3" text="Need help?"></TextBox>
      <Spacer size={16} />

      <RichText
        spans={[
          <span key={0}>
            Give us a call using the number listed on the back of your Member ID
            card or{' '}
          </span>,
          <span className="link" key={1}>
            <a href={process.env.NEXT_PUBLIC_PORTAL_CONTACT_US_URL ?? ''}>
              contact us
            </a>
          </span>,
        ]}
      />
    </div>
  );
};
