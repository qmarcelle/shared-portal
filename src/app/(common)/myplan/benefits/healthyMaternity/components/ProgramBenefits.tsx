/* eslint-disable @typescript-eslint/no-unused-vars */
/* eslint-disable react/jsx-key */
import { Column } from '@/components/foundation/Column';
import { Header } from '@/components/foundation/Header';
import { Row } from '@/components/foundation/Row';
import { Spacer } from '@/components/foundation/Spacer';
import { IComponent } from '../../../components/IComponent';
import { TextBox } from '../../../components/foundation/TextBox';
import { BenefitDetails } from '../models/benefit_details';

interface ProgramBenefitsProps extends IComponent {
  benefits: BenefitDetails[];
}

export const ProgramBenefits = ({ benefits }: ProgramBenefitsProps) => {
  return (
    <>
      <Spacer size={12} />
      <Header className="title-2 px-3" text="Program Benefits"></Header>
      <Spacer size={25} />
      <Row className="hidden sm:flex px-3">
        {benefits.slice(0, benefits.length).map((item, index) => (
          <Column className="md:pr-4 md:w-1/3">
            {item.benefitIcon}
            <Spacer size={5} />
            <TextBox text={item.benefitLabel} className="font-bold" />
            <Spacer size={5} />
            <TextBox text={item.benefitCopy} />
          </Column>
        ))}
      </Row>
      <Column className="block sm:hidden px-3">
        {benefits.slice(0, benefits.length).map((item, index) => (
          <Column className="pr-4 mb-4">
            {item.benefitIcon}
            <Spacer size={5} />
            <TextBox text={item.benefitLabel} className="font-bold" />
            <Spacer size={5} />
            <TextBox text={item.benefitCopy} />
          </Column>
        ))}
      </Column>
    </>
  );
};
