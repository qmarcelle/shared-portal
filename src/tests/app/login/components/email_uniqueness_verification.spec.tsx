import { EmailUniquenessVerification } from '@/app/login/components/EmailUniquenessVerification';
import '@testing-library/jest-dom';
import { fireEvent, render, screen, waitFor } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
const mockReplace = jest.fn();
jest.mock('next/navigation', () => ({
  useRouter() {
    return {
      prefetch: () => null,
      replace: mockReplace,
    };
  },
}));
const setupUI = () => {
  return render(<EmailUniquenessVerification />);
};
describe('Email Unique Verification screen', () => {
  test('Email Unique Verification screen UI', async () => {
    const component = setupUI();

    expect(
      screen.getByRole('heading', { name: 'Choose Your Email Address' }),
    ).toBeVisible();
    expect(
      screen.getByText(
        'Your email is invalid, or the original email address you provided is already associated with another account. Please provide your date of birth and new email address to continue.',
      ),
    ).toBeVisible();
    //  expect(screen.getByLabelText(/Email Address/i)).toBeVisible();
    expect(screen.getByLabelText(/Confirm Email Address/i)).toBeVisible();
    expect(screen.getByRole('button', { name: /Next/i })).toBeVisible();

    expect(
      screen.getByText(
        'Give us a call using the number listed on the back of your Member ID card or',
      ),
    ).toBeVisible();

    expect(component.baseElement).toMatchSnapshot();
  });

  test('Email Unique Verification screen UI Field Validations', async () => {
    const component = setupUI();
    const dateEntryInput = screen.getByLabelText('Date of Birth (MM/DD/YYYY)');

    //check for invalid date formate
    await userEvent.type(dateEntryInput, '18/11/1970');
    await waitFor(() => {
      expect(
        screen.getByText('Please follow the MM/DD/YYYY format.'),
      ).toBeVisible();
    });
    //check for leap year date
    await userEvent.type(dateEntryInput, '02/29/2023');
    fireEvent.click(screen.getByRole('button', { name: /Next/i }));
    await waitFor(() => {
      expect(
        screen.getByText('Please follow the MM/DD/YYYY format.'),
      ).toBeVisible();
    });

    //check valid email formate
    const emailEntryInput = screen.getByLabelText('Email Address');
    await userEvent.type(emailEntryInput, 'chall123gmail.com');
    await waitFor(() => {
      expect(
        screen.getByText('Please enter a valid Email address.'),
      ).toBeVisible();
    });

    //check confirm email address
    const confirmEmailEntryInput = screen.getByLabelText(
      'Confirm Email Address',
    );
    await userEvent.type(emailEntryInput, 'chall123@gmail.com');
    await userEvent.type(confirmEmailEntryInput, 'chall1811@gmail.com');
    await waitFor(() => {
      expect(
        screen.getByText(
          'The email addresses must match. Please check and try again.',
        ),
      ).toBeVisible();
    });

    expect(component.baseElement).toMatchSnapshot();
  });
});
