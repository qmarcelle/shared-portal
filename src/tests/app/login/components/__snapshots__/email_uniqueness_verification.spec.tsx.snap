// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Email Unique Verification screen Email Unique Verification screen UI 1`] = `
<body>
  <div>
    <section
      id="mainSection"
    >
      <h1
        class="title-1 title-2"
      >
        Choose Your Email Address
      </h1>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
      />
      <p
        class="body-1"
      >
        Your email is invalid, or the original email address you provided is already associated with another account. Please provide your date of birth and new email address to continue.
      </p>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col w-full text-field"
      >
        <p>
          Date of Birth (MM/DD/YYYY)
        </p>
        <div
          class="flex flex-row items-center input  "
        >
          <input
            aria-label="Date of Birth (MM/DD/YYYY)"
            class=""
            maxlength="10"
            type="text"
            value=""
          />
          <div
            class="cursor-pointer"
          />
        </div>
        <div
          class="error-container mt-1 p-1"
        >
          <div />
        </div>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col w-full text-field"
      >
        <p>
          Email Address
        </p>
        <div
          class="flex flex-row items-center input  "
        >
          <input
            aria-label="Email Address"
            class=""
            type="email"
            value=""
          />
          <div
            class="cursor-pointer"
          />
        </div>
        <div
          class="error-container mt-1 p-1"
        >
          <div />
        </div>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col w-full text-field"
      >
        <p>
          Confirm Email Address
        </p>
        <div
          class="flex flex-row items-center input  "
        >
          <input
            aria-label="Confirm Email Address"
            class=""
            type="email"
            value=""
          />
          <div
            class="cursor-pointer"
          />
        </div>
        <div
          class="error-container mt-1 p-1"
        >
          <div />
        </div>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-row justify-center items-center tooltip"
      >
        <span
          class="tooltiptext !p-2"
        >
          Fill out the form to continue.
        </span>
        <button
          aria-label="Next"
          class="button-text primary text-center flex flex-row justify-center items-center min-w-fit inactive undefined"
          type="button"
        >
          Next
        </button>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class=""
        style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
      />
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <p
        class="body-1 title-3"
      >
        Need help?
      </p>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
      />
      <span
        class="body-1 undefined"
      >
        <span>
          Give us a call using the number listed on the back of your Member ID card or
           
        </span>
        <span
          class="link"
        >
          <a
            href=""
          >
            contact us
          </a>
        </span>
      </span>
    </section>
  </div>
</body>
`;

exports[`Email Unique Verification screen Email Unique Verification screen UI Field Validations 1`] = `
<body>
  <div>
    <section
      id="mainSection"
    >
      <h1
        class="title-1 title-2"
      >
        Choose Your Email Address
      </h1>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
      />
      <p
        class="body-1"
      >
        Your email is invalid, or the original email address you provided is already associated with another account. Please provide your date of birth and new email address to continue.
      </p>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col w-full text-field"
      >
        <p>
          Date of Birth (MM/DD/YYYY)
        </p>
        <div
          class="flex flex-row items-center input  error-input"
        >
          <input
            aria-label="Date of Birth (MM/DD/YYYY)"
            class=""
            maxlength="10"
            type="text"
            value="18/11/1970"
          />
          <div
            class="cursor-pointer"
          />
        </div>
        <div
          class="error-container mt-1 p-1"
        >
          <div>
            <div
              class="flex flex-row"
            >
              <div
                class="inline-flex"
              >
                <img
                  alt="ErrorIcon"
                  class="icon"
                  data-nimg="1"
                  decoding="async"
                  height="40"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                  style="color: transparent;"
                  width="40"
                />
                <p
                  class="ml-2"
                >
                  Please follow the MM/DD/YYYY format.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col w-full text-field"
      >
        <p>
          Email Address
        </p>
        <div
          class="flex flex-row items-center input  "
        >
          <input
            aria-label="Email Address"
            class=""
            type="email"
            value="chall123gmail.comchall123@gmail.com"
          />
          <div
            class="cursor-pointer"
          />
        </div>
        <div
          class="error-container mt-1 p-1"
        >
          <div />
        </div>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-col w-full text-field"
      >
        <p>
          Confirm Email Address
        </p>
        <div
          class="flex flex-row items-center input input-focus error-input"
        >
          <input
            aria-label="Confirm Email Address"
            class=""
            type="email"
            value="chall1811@gmail.com"
          />
          <div
            class="cursor-pointer"
          />
        </div>
        <div
          class="error-container mt-1 p-1"
        >
          <div>
            <div
              class="flex flex-row"
            >
              <div
                class="inline-flex"
              >
                <img
                  alt="ErrorIcon"
                  class="icon"
                  data-nimg="1"
                  decoding="async"
                  height="40"
                  loading="lazy"
                  src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
                  srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
                  style="color: transparent;"
                  width="40"
                />
                <p
                  class="ml-2"
                >
                  The email addresses must match. Please check and try again.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class="flex flex-row justify-center items-center tooltip"
      >
        <span
          class="tooltiptext !p-2"
        >
          Fill out the form to continue.
        </span>
        <button
          aria-label="Next"
          class="button-text primary text-center flex flex-row justify-center items-center min-w-fit inactive undefined"
          type="button"
        >
          Next
        </button>
      </div>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <div
        class=""
        style="width: 100%; height: 1px; background-color: rgb(204, 204, 204);"
      />
      <span
        style="display: block; width: 1px; min-width: 1px; height: 32px; min-height: 32px;"
      />
      <p
        class="body-1 title-3"
      >
        Need help?
      </p>
      <span
        style="display: block; width: 1px; min-width: 1px; height: 16px; min-height: 16px;"
      />
      <span
        class="body-1 undefined"
      >
        <span>
          Give us a call using the number listed on the back of your Member ID card or
           
        </span>
        <span
          class="link"
        >
          <a
            href=""
          >
            contact us
          </a>
        </span>
      </span>
    </section>
  </div>
</body>
`;
